docs/porting/functionality-differences.md

```diff
@@ -919,7 +918,7 @@ Older and less common usages of `EVP_PKEY` have been removed. For example, signi
   <td rowspan=2>
   <p>
     <span>
-        <a href="https://github.com/aws/aws-lc/blob/5c358103c5df836b9343bf995717b5bc13d5e82f/include/openssl/digest.h#L303-L310">
+        <a href="https://github.com/aws/aws-lc/blob/746d06505b3a3827cf61959ca0c3d87c3f21accc/include/openssl/digest.h#L347-L370">
             digest.h
```

Comment:
This link highlight includes the SHA3 flag which looks unrelated, or should this table include EVP_MD_unstable_sha3_* functions which do nothing?